privaterelay/plans.py

```diff
@@ -18,8 +18,8 @@
 They all return a PlanCountryLangMapping dict, which has this structure:
```

Comment:
question (non-blocking): is there a reason (other than just historical?) we call this `PlanCountryLangMapping` when this structure seems to be more accurately called `CountryLangPlanMapping`?

Reply:
Yep, historical. These structures were previously in settings:

https://github.com/mozilla/fx-private-relay/blob/085142c4148e607a009931f454e1a854c499a68f/privaterelay/settings.py#L662
https://github.com/mozilla/fx-private-relay/blob/085142c4148e607a009931f454e1a854c499a68f/privaterelay/settings.py#L796
https://github.com/mozilla/fx-private-relay/blob/085142c4148e607a009931f454e1a854c499a68f/privaterelay/settings.py#L818

So the structure type was named after the common element from the settings names. The "Plan" is premium, phones, or bundle.

The structure is:

* country code
    * language code
        * term (monthly, yearly)
            * price details
            
so maybe `CountryLanguageTermPriceMapping`? `CountryLangTermPriceMappingForPlan`? `PlanMap`? I give up.

Reply:
No worries - we don't need to change it.